feat(cli): output selector (#1419)
iotaledger/iota-sdk: cli/CHANGELOG.md

### Security -->

## 1.2.0 - 2023-xx-xx

### Changed

> - `AccountCommand::Output` now accepts either an index or an ID;

Reviewer: ```suggestion
- `AccountCommand::Output` now accepts either a list index or an `OutputId`;
```

### Security -->

## 1.2.0 - 2023-xx-xx

### Changed

- `AccountCommand::Output` now accepts either a list index or an `OutputId`;

> - `output` and `outputs` now order outputs by their corresponding booked milestone timestamp;

Reviewer: `output` just shows one, it doesn't order anything

Author: ```rust
OutputSelector::Index(index) => {
            let mut outputs = account.outputs(None).await?;
            outputs.sort_by(|a, b| {
                a.metadata
                    .milestone_timestamp_booked()
                    .cmp(&b.metadata.milestone_timestamp_booked())
            });
            outputs.into_iter().rev().nth(index)
        }
```

Reviewer: Still, just shows one so the comment itself doesn't really make sense

Author: didn't know the changelog can't refer to the logic of a function...the function itself orders the outputs given an index.

Reviewer: I think it reads a bit weird but if you want to keep it, fine

Author: who's going to read (and trying to understand) it anyway? :trollface:

### Security -->

## 1.2.0 - 2023-xx-xx

### Changed

- `AccountCommand::Output` now accepts either a list index or an `OutputId`;
- `output`, `outputs`, and `unspent_outputs` now order outputs by their corresponding booked milestone timestamp;

> - `outputs` and `unspent_outputs` now include spent/unspent information;

Reviewer: Move to added section

Reviewer: Also date is new to outputs

> - `outputs` and `transactions` now add the suffix `UTC` to the formatted date string;

Reviewer: Same

### Security -->

## 1.2.0 - 2023-xx-xx

### Added

> - `outputs` and `unspent_outputs` now print the booked milestone timestamps and sort by them;

Reviewer: ```suggestion
- `outputs` and `unspent_outputs` print the booked milestone timestamps and sort by them;
```

> - `outputs` and `unspent_outputs` now print a list of outputs ordered by their corresponding booked milestone timestamp;

Reviewer: ```suggestion
- `outputs` and `unspent_outputs` now print the booked milestone timestamps and sort by them;
```

> - `outputs` and `unspent_outputs` now include spent/unspent information;

Reviewer: ```suggestion
- `outputs` and `unspent_outputs` include spent/unspent information;
```

> - `transactions` now adds the suffix `UTC` to the generated formatted date string;

Reviewer: ```suggestion
- `UTC` suffix to the formatted date of `transactions`;
```

### Changed

> - `AccountCommand::Output` now accepts either a list index or an `OutputId`;

Reviewer: ```suggestion
- `AccountCommand::Output` accepts either a list index or an `OutputId`;
```

> - `transactions` now adds the suffix `UTC` to the generated formatted date string;

Reviewer: Put in Added

Author: Done
